.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
@SMK1085 we are already using EditorConfig for such formatting settings. It looks like the VS should support it out-of-the-box: http://editorconfig.org/

It may look as a minor thing but lets make sure we keep the formatting consistent.